tck/features/ProcedureCallAcceptance.feature

```diff
@@ -0,0 +1,515 @@
+#
+# Copyright (c) 2002-2016 "Neo Technology,"
+# Network Engine for Objects in Lund AB [http://neotechnology.com]
+#
+# This file is part of Neo4j.
+#
+# Neo4j is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+
+Feature: ProcedureCallAcceptance
+
+  Background:
+    Given an empty graph
+
+  Scenario: In-query call to procedure that takes arguments fails when trying to pass them implicitly
+    And there exists a procedure test.my.proc(in :: INTEGER?) :: (out :: INTEGER?):
+      | in | out |
+    When executing query:
+    """
+    CALL test.my.proc YIELD out
+    RETURN out
+    """
```

Comment:
Indentation of the query strings is off. Should be indented two spaces in from the `When` step.